doc/script_commands.txt

```diff
@@ -1904,57 +1904,61 @@ will result in error and termination of the script.
 ---------------------------------------
 
 *function <function name>;
-*<function name>{(<argument>, ...<argument>)};
 *function <function name> {
```

Comment:
something here bothers me: we're using curly braces to denote optional arguments, but in this case literal curly braces are required; how should this be written? escaped curly braces (`\{ <code> \}`) or a custom sequence like we do for `%TAB%`?

Reply:
I think escaping them might cause more confusion, so I'd advise against that (or someone will really try to write it as `\{` in scripts).

I'd personally leave it as is (I don't believe it's ambiguous because it's not a function argument). Another option would be `%LCURLY%` / `%RCURLY%` but considering that I already find the `%TAB%` annoyingly hard to read...